src/Console/transformer.stub

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace DummyNamespace;
+
+use DummyModel;
+use Rexlabs\Laravel\Smokescreen\Transformers\AbstractTransformer;
+
+/**
+ * The DummyModelName transformer.
+ *
+ */
+class DummyClass extends AbstractTransformer
```

Comment:
It would be cool if this was a `TransformerStub.blade.php` file that you could optionally publish. 
This would give the user full control over the generated result, and ultimately make it easier for you to implement the reflection properties.